### PR TITLE
docs: extend skip rules example

### PR DESCRIPTION
**Pull request by JStickler**
Fleshing this out a little, because for some reason I did NOT understand how to do this when I first read it and just copy/pasted the example.  Which of course, isn't how it works....

### CODE REVIEW

File: docs/sources/review/lint-prose/_index.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Repositories that run Vale in CI include:
 
 ## Skip rules
 
-To skip a rule, enclose the section with HTML comments that first disable, and then re-enable the specific Vale rule.
+To skip a rule, enclose the section with HTML comments that first disable, and then re-enable the specific Vale rule. Include the specific rule name in the comment, for example `Grafana.We` or `Grafana.Google.Ellipses`. The `Grafana.Quotes` rule is a made up example rule to illustrate how to disable a rule.
 
 To disable the `Grafana.Quotes` rule:
```
Comment (jdbaldry) [inline docs/sources/review/lint-prose/_index.md:31]: Fixing the rule name for [`Grafana/GoogleEllipses`](https://github.com/grafana/writers-toolkit/blob/main/vale/Grafana/GoogleEllipses.yml)

```suggestion
To skip a rule, enclose the section with HTML comments that first disable, and then re-enable the specific Vale rule.
Include the specific rule name in the comment, for example `Grafana.We` or `Grafana.GoogleEllipses`. 
The `Grafana.Quotes` rule is a made up example rule to illustrate how to disable a rule.

To disable the `Grafana.Quotes` rule:
```

**Comment by jdbaldry** [comment]
Thank you for the contribution!

File: docs/sources/review/lint-prose/_index.md (hunk 1 above)
Comment (clayton-cornell) [reply docs/sources/review/lint-prose/_index.md:31]: Why use a "fake" rule and have to explain it's fake? Can't we illustrate this with a real rule?

Comment (jdbaldry) [reply docs/sources/review/lint-prose/_index.md:31]: That makes sense to me. It seems that this was a typo of [`Grafana.GoogleQuotes`](https://github.com/grafana/writers-toolkit/blob/main/vale/google.jsonnet#L31) that has been removed.

Comment (jdbaldry) [reply docs/sources/review/lint-prose/_index.md:31]: I guess it shouldn't matter really if the rule is real or not after we have the explanation that you need to include the name of the rule you want to disable:

```suggestion
To skip a rule, enclose the section with HTML comments that first disable, and then re-enable the specific Vale rule. Include the specific rule name in the comment, for example `Grafana.We` or `Grafana.Google.Ellipses`.

To disable the `Grafana.Quotes` rule:
```

Comment (jdbaldry) [reply docs/sources/review/lint-prose/_index.md]: I've reopened this conversation because I think it's unresolved.

Comment (JStickler) [reply docs/sources/review/lint-prose/_index.md]: @clayton-cornell If you've got a real example, I'm happy to include it.  I've only skipped a rule once and I didn't think it was a great example to publish.

Comment (clayton-cornell) [reply docs/sources/review/lint-prose/_index.md]: There are several in https://raw.githubusercontent.com/grafana/writers-toolkit/main/docs/sources/write/shortcodes/index.md (and other places).... `<!-- vale Grafana.GooglePassive = NO -->` for example.

**Review by chri2547 — APPROVED** [review]
LGTM! Thank you!

**Review by jdbaldry — APPROVED** [review]
LGTM, thanks for updating the example.

**Pull request merged**